src/idl_gen_python.cpp

```diff
@@ -144,6 +151,30 @@ class PythonGenerator : public BaseGenerator {
     code += Indent + Indent + "return x\n";
     code += "\n";
   }
+  // Checks for propper file identifier
+  void HasFileIdentifier(const StructDef &struct_def,
```

Comment:
What C++ should do?
Looks like a generator not as the statement `Has a property`.
Do you want to generate `esapedID` string based on the `parser_.file_identifier_`?
Why not use the std::string and std::stringstream for this without `sprintf`?

Reply:
Thanks for the suggestion. ive implemented it.

https://github.com/pyottamus/flatbuffers/blob/618c9bf64d275781288fd06b62a8299ac964cb39/src/idl_gen_python.cpp#L555-L576